### PR TITLE
Removes fires from depot

### PR DESCRIPTION
**Pull request by Kyep**
## What Does This PR Do
1) The depot reactor self destruct no longer spreads fire.
2) Sources of plasma have been removed from the depot. This includes tanks of plasma, a plasma airlock, and plasma floor tiles on the east side of the depot.
3) N2O gas grenade has been removed from depot armory loot table.

## Why It's Good For The Game
Recently, we have been having some issues where by the MC (master controller, a crucial piece of game code) would crash while atmos stuff was going on in the depot.
This series of changes is intended to prevent atmos stuff in the depot from affecting the stability of the server overall.

## Images of changes
Before:
depot_tank_changes

After:
depot_new
The two smaller red-boxed areas in this image have had plasma tanks removed from them.
The larger red-boxed area on the right has had a plasma tank, plasma airlock, plasma floor tiles, and falsewall spawners removed.

## Changelog
:cl: Kyep
fix: Removed fire-based hazards from depot, for the sake of server stability.
/:cl:

### CODE REVIEW

File: code/game/objects/effects/spawners/random_spawners.dm [hunk 1]
```diff
@@ -304,11 +303,6 @@
 		/turf/simulated/wall/mineral/plastitanium/nodiagonal = 2,
 		/obj/structure/falsewall/plastitanium = 2)
 
-/obj/effect/spawner/random_spawners/syndicate/layout/door/secret
-	name = "50pc falsewall 50pc wall"
-	result = list(/turf/simulated/wall/mineral/plastitanium/nodiagonal = 1,
-		/obj/structure/falsewall/plastitanium = 1)
-
```
Comment (AffectedArc07) [inline code/game/objects/effects/spawners/random_spawners.dm:311]: Why did you remove this falsewall spawner?

Comment (Kyep) [reply code/game/objects/effects/spawners/random_spawners.dm:311]: Because it isn't used anymore. The only place it was used was in the red-boxed area on the right, around the (now also removed) plasma tank. Those tiles are solid walls now. The object was only defined so it could be used there. Not it isn't used there, it has no justification to exist.

**Comment by Fox-McCloud** [comment]
I question if this will fix the issue, but /shrug

**Comment by datlo** [comment]
Chameleon flags in the syndi depot are booby trapped with plasmafire grenades. I suggest replacing it with a syndi minibomb, should be a fairly simple edit in the chameleon flag file

**Comment by Kyep** [comment]
@datlo My suspicion is that it is the overload (self destruct), not fires in general, that cause the MC issues. All the recent examples of MC issues have been at times the overload was active. So, I really don't think that the flags are the cause. If, however, I see further issues, especially after flags are used, then I will go after the flags as well.
